bioregistry.ts

```diff
@@ -46,39 +46,39 @@ export const bioregistryRules = [
     onMatch,
   },
   {
-    regex: /(biocyc:[A-Z-0-9]+(\:)?[A-Za-z0-9+_.%-:]+)/,
+    regex: /(biocyc:[A-Z-0-9]+(?:\:)?[A-Za-z0-9+_.%-:]+)/,
     onMatch,
   },
   {
     regex: /(biogrid:\d+)/,
     onMatch,
   },
   {
-    regex: /(bioportal:(?:(\d+)|(\w+)))/,
+    regex: /(bioportal:(?:\d+)|(?:\w+))/,
```

Comment:
@allaway : It looks like the latest change has caused a regression.  These inner groups are no longer grouped.
For example:
/(bioportal:(?:\d+)|(?:\w+))/
should be 
/(bioportal:(?:(?:\d+)|(?:\w+)))/

and
/(ncbitaxon:(?:\d+)|(?:[a-zA-Z_]+))/
should be
/(ncbitaxon:(?:(?:\d+)|(?:[a-zA-Z_]+)))/

Reply:
@jay-hodgson , this should be fixed. Thanks for catching this, it wasn't as simple of a mistake as I thought it was!